File: mixedProblems/problem_8.py
# Problem 8 : How do you swap two numbers without using the third variable?

firstNumber = int(input("Enter First Number: "))
secondNumber = int(input("Enter Second Number: "))

afterSwapFirstNumber = (firstNumber+secondNumber)-firstNumber
afterSwapSecondNumber = (firstNumber+secondNumber)-secondNumber

print(f'After Swap: First Number is {afterSwapFirstNumber} and Second Number is {afterSwapSecondNumber}')

File: mixedProblems/problem_8.js
// Problem 8 : How do you swap two numbers without using the third variable?

var firstNumber = 10
var secondNumber = 20

var afterSwapSecondNumber = (firstNumber+secondNumber)-secondNumber
var afterSwapFirstNumber = (firstNumber+secondNumber)-firstNumber

console.log(`After Swapping: First number is ${afterSwapFirstNumber} and Second Number is ${afterSwapSecondNumber}`)